gm_dotnet_managed/GmodNET/GameConsoleWriter.cs

```diff
@@ -0,0 +1,20 @@
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+
+/// <summary>
+/// Redirects Console's output to game's console
+/// </summary>
+public class GameConsoleWriter : TextWriter
+{
+    public override string NewLine { get => "\n"; }
```

Comment:
I would rather prefer simpler syntax for get only property:
```csharp
public override string NewLine => "\n";
```

Reply:
resolved in https://github.com/GmodNET/GmodDotNet/pull/70/commits/97684fd20c56f385941388104826e26a2dee5bf7